Kommunicate-io/Kommunicate-Web-SDK
# FWH-442 The web storage data should be stored as per application id

### What do you want to achieve?
- the web storage  data would be stored as per application id

### PR Checklist


### How was the code tested?

- open incognito window
- create a widget instance using old build
- check cookies, session storage and local storage for km related data
- close the window directly( without logout)
- deploy this branch and then open the widget instance
- check cookies, session storage and local storage for km related data it should be updated

- repeat the test for exiting instances 
- try different browsers 

### What new thing you came across while writing this code? 
-

### In case you fixed a bug then please describe the root cause of it? 
-

### Screenshot
- 

NOTE: Make sure you're comparing your branch with the correct base branch
